packages/swc-helpers/esm/_update.js

```diff
@@ -1,6 +1,7 @@
-import _get from "./_get.mjs";
-import _set from "./_set.mjs";
-export default function _update(target, property, receiver, isStrict) {
+import { _get } from "./_get.js";
```

Comment:
Will this work? Isn't the export named `_`?

Reply:
Yes there are two named export.